packages/block-editor/src/components/inner-blocks/use-inner-block-template-sync.js

```diff
@@ -81,6 +82,7 @@ export default function useInnerBlockTemplateSync(
 			);
 
 			if ( ! isEqual( nextBlocks, currentInnerBlocks ) ) {
+				__unstableMarkNextChangeAsNotPersistent();
```

Comment:
I'm not sure if patching the `useInnerBlockTemplateSync` hook for a single use case is a good idea 😅

Reply:
It's not only this, we've hit this before: see https://github.com/WordPress/gutenberg/pull/45843

Reply:
Can we remove this from this PR and wait on the decision in https://github.com/WordPress/gutenberg/pull/45843?

Reply:
I'll try to rebase on that or something. b/c otherwise this PR does not work.